lib/utils/constants.dart

```diff
@@ -2,6 +2,7 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 
 // Images ->
+const String Back = "assets/back.png";
 const String SochemIcon = "assets/sochem.jpeg";
 const String ExampleProfile = "assets/sampleprofile.jpeg";
```

Comment:
BackIcon will be a better name

Reply:
Should we instead of adding an additional icon, just use some custom icon provided by Flutter? What's your view?

Reply:
Yeah that is preffered